src/pools/selection-strategies.ts

```diff
@@ -54,14 +54,71 @@ class RoundRobinWorkerChoiceStrategy<Worker extends IWorker, Data, Response>
   }
 }
 
+/**
+ * Selects the less recently used worker.
+ *
+ * @template Worker Type of worker which manages the strategy.
+ * @template Data Type of data sent to the worker. This can only be serializable data.
+ * @template Response Type of response of execution. This can only be serializable data.
+ */
+class LessRecentlyUsedWorkerChoiceStrategy<
+  Worker extends IWorker,
+  Data,
+  Response
+> implements IWorkerChoiceStrategy<Worker> {
+  /**
+   * @param pool The pool instance.
+   */
+  constructor (private pool: IPoolInternal<Worker, Data, Response>) {
+    this.pool = pool
+  }
+
+  /** @inheritdoc */
+  public choose (): Worker {
+    let minNumberOfTasks = Infinity
+    let lessRecentlyUsedWorker: Worker = {} as Worker
+    for (const [worker, numberOfTasks] of this.pool.tasks) {
+      if (numberOfTasks === 0) {
+        lessRecentlyUsedWorker = worker
+        break
+      } else if (numberOfTasks < minNumberOfTasks) {
+        minNumberOfTasks = numberOfTasks
+        lessRecentlyUsedWorker = worker
```

Comment:
same as above

Reply:
But here I can't, the loop need to iterate over all the map to elect the less recently used.

Reply:
Make sense, it's ok